apps/condo/domains/ticket/schema/ExportTicketsService.js

```diff
@@ -63,7 +66,7 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
                         operator: ticket.operator || ticket.createdBy || '',
                         executor: ticket.executor || '',
                         assignee: ticket.assignee || '',
-                        comments: ticket.TicketComment.join('\n'),
+                        comments: comments.map(comment => comment.split(':').pop()).join(TICKET_COMMENTS_SEPARATOR),
```

Comment:
as far as I know pop returns either last element of an array or undefined, so `.join(...)` here will do nothing...